app/AuthContext.tsx

```diff
@@ -0,0 +1,54 @@
+"use client"
```

Comment:
Where is this context being used? I didn't see AuthContextProvider being injected into the main layout / page

Reply:
Added